include/fields2cover/route_planning/route_planner_base.h

```diff
@@ -21,7 +21,7 @@
 namespace f2c::rp {
 
 class RoutePlannerBase {
- public:
```

Comment:
public, private and protected words should only have one space more than the class name (as before)